app/src/vuex/modules/user.js

```diff
@@ -33,9 +33,11 @@ export default () => {
         try {
           await Auth.signInWithEmailLink(user.email, window.location.href)
           await Auth.onAuthStateChanged((user) => {
```

Comment:
this is a global listener which you only need to call once. whenever the user signs in or out, this function will be called. I would even assume that with this logic, the code is called multiple times.